release/pyproject.toml

```diff
@@ -11,3 +11,67 @@ dev = [
     "pip==24.0",
     "pip-tools==7.4.1"  # To add hashes to requirements
 ]
+
+[tool.bumpversion]
+current_version = "5.12.0-rc.1"
+parse = """(?x)
+    (?P<major>0|[1-9]\\d*)\\.
+    (?P<minor>0|[1-9]\\d*)\\.
+    (?P<patch>0|[1-9]\\d*)
+    (?:
+        -                                    # dash separator for pre-release section
+        (?P<pre_release_label>[a-zA-Z-]+)    # pre-release label
+        \\.
+        (?P<pre_release_number>0|[1-9]\\d*)  # pre-release version number
+    )?                                       # pre-release section is optional
+"""
+serialize = [
+    "{major}.{minor}.{patch}-{pre_release_label}.{pre_release_number}",
+    "{major}.{minor}.{patch}",
+]
+commit = true
+tag = true
+
+[tool.bumpversion.parts.pre_release_label]
+values = ["rc", "final"]
+optional_value = "final"
```

Comment:
Does "optional" mean that it's the default?

Reply:
See previous comment.